Include/internal/pycore_global_objects.h

```diff
@@ -34,6 +34,24 @@ extern "C" {
     }
 
 
+/* bytes objects */
+
+#define _PyBytes_EMPTY_INIT \
```

Comment:
This is used exactly once. Remove it.

Reply:
I added it so the code tied to the layout of `PyBytesObject` would be in one spot, making it easier to find than if it were embedded in the larger initializer.

Reply:
I sorted this out.